test/sewing_test.c

```diff
@@ -158,17 +158,12 @@ void Sew_Test_Set_hwloc_thread_affinity
                             !hwloc_set_thread_cpubind
                             (
                                 t
-                                , threads[threads_left - 1]
+                                , threads[threads_left - 1].handle
```

Comment:
That line won't work as it'll break the Linux build (which isn't a struct).

I'm loath to wrap it inside an `#ifdef`. And I'm not going to change the API just for that example. 

Ok, I give up, just wrap that in an `#ifdef` block.
```
#if __linux__
#elif defined(_WIN32)
#else
#error
#endif
```